src/CommonLib/LDAPConfig.cs

```diff
@@ -8,14 +8,25 @@ public class LDAPConfig
         public string Password { get; set; } = null;
         public string Server { get; set; } = null;
         public int Port { get; set; } = 0;
-        public bool SSL { get; set; } = false;
+        public bool ForceSSL { get; set; } = false;
```

Comment:
Will this change the name of the configuration field for this in the JSON?

Reply:
We don't save this configuration anywhere, so its not an issue. This is a purely runtime config